city_scrapers/spiders/alle_county.py

```diff
@@ -0,0 +1,159 @@
+from city_scrapers.spider import Spider
+
+
+class AlleCountySpider(Spider):
+    name = 'alle_county'
+    agency_name = 'Allegheny County Government'
+    timezone = 'America/New_York'
+    allowed_domains = ['alleghenycounty.legistar.com']
+    START_URL = 'https://alleghenycounty.legistar.com'
+    start_urls = [START_URL]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` a dict that follows
+        the `Open Civic Data event standard
+        <http://docs.opencivicdata.org/en/latest/data/event.html>`.
+
+        Change the `_parse_id`, `_parse_name`, etc methods
+        to fit your scraping needs.
+        """
+        events = self._make_legistar_call()
+        return self._parse_events(events)
+
+    def _make_legistar_call(self, since=None):
+        urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+        les = LegistarEventsScraper(requests_per_minute=0)
+        les.EVENTSPAGE = self.START_URL + '/Calendar.aspx'
+        les.BASE_URL = self.START_URL
+        if not since:
+            since = datetime.today().year
+        return les.events(since=since)
+
+    def _parse_events(self, events):
+        for item, _ in events:
+            data = {
+                'timezone': self.timezone,
+                '_type': 'event',
+                'name': self._parse_name(item),
+                'event_description': '',
+                'all_day': self._parse_all_day(item),
+                'classification': self._parse_classification(item),
+                'start': self._parse_start(item),
+                'end': self._parse_end(item),
+                'location': self._parse_location(item),
+                'documents': self._parse_documents(item),
+                'sources': self._parse_sources(item),
+            }
+            data['id'] = self._generate_id(data)
+            data['status'] = self._generate_status(data,
+                                                   item['Meeting Location'])
+            yield data
+
+    def _parse_location(self, item):
+        """
+        Parse or generate location.
+        """
+        Room = item.get('Meeting Location', None)
+        if Room:
+            Room = Room + ','
+
+        return {
+            'address': "{Room} {Location}"
+            .format(Room=Room, Location=('436 Grant Street, '
+                                         'Pittsburgh, PA 15219')),
+            'name': '',
+            'neighborhood': ''
+        }
+
+    def _parse_all_day(self, item):
+        """
+        Parse or generate all-day status. Defaults to false.
+        """
+        return False
+
+    def _parse_name(self, item):
+        """
+        Parse or generate event name.
+        """
+        return item['Name']['label']
+
+    def _parse_classification(self, item):
+        """
+        Differentiate board meetings from public hearings.
+        """
+        if 'hearing' in item['Meeting Location'].lower():
```

Comment:
One last thing I missed on my first pass, this should also return `COMMITTEE` if 'Committee' is in the meeting name and default to `CITY_COUNCIL` instead of `BOARD` (even though it's not a city we're using that to generally describe legislative bodies). The check for `hearing` is good as-is. Sorry for missing that!

Reply:
np. Done.